### PR TITLE
Added 59 new icons

### PR DESCRIPTION
**Pull request by The-King-Slayer**
## Pull Request Checklist:

* [x] Have you added an explanation of which icons you've added and why you'd like us to include them?
* [x] Have you created three versions of each icon across the `svgs`, `appstract-light`, and `appstract-dark` folders?
* [x] Do all three icons have the same name, except for the file type ending? (For example, `app.svg`, `app.png`, and `app.png`)
* [x] Is each file name only comprised of lowercase letters, numbers, and underscores? Underscores should be used only to replace spaces. File names may not begin with a number.
    - Acceptable names: `hello.svg`, `bye_bye.svg`, `disney_plus.svg`, `numbers_123.svg`
    - Unacceptable names: `spaces everywhere.svg`, `YouTube.svg`, `$%*&^.svg`, `dots.between.svg`, `1something.svg`.
* [x] Have you ensured that all svgs and pngs are 192x192 px with a line width of 6px and a transparent background?
* [x] Did you add activity names for all of your icons to `app/xml/appfilter.xml`?
* [x] Did you add yourself to the `app/xml/contributors.xml` file? :)

### Template

For each icon added, please add an entry to the pull request (this text) formatted like this:

* App Name (`app_name.svg`) - [New/Redesigned] Icon
```
<item component="ComponentInfo{Activity Name 1}" drawable="app_name"/>
<item component="ComponentInfo{Activity Name 2}" drawable="app_name"/>
```

###### Gotchas:
- The name inside `drawable=` must match the name of the file (For example, `orange.svg` would have `drawable="orange"`)
- For help finding/adding activity names, please see the [Contributing Guide](https://github.com/mirrorkeydev/Appstract/blob/master/CONTRIBUTING.md#activity-names)

### Examples (delete before opening PR)

* Camera (`camera.svg`) - Redesigned Icon
```
<item component="ComponentInfo{com.android.camera/com.android.camera.view.CameraActivity}" drawable="camera"/>
<item component="ComponentInfo{com.mediatek.camera/com.mediatek.camera.CameraLauncher}" drawable="camera"/>
<item component="ComponentInfo{com.GoogleCamera.Wichaya/com.android.camera.CameraLauncher}" drawable="camera"/>
<item component="ComponentInfo{com.hmdglobal.app.camera/com.hmdglobal.app.camera.GuideActivity}" drawable="camera"/>
```

* Poshmark (`poshmark.svg`) - New Icon
```
<item component="ComponentInfo{com.poshmark.app/com.poshmark.ui.MainActivity}" drawable="poshmark"/>
```

* Disney+ (`disney_plus.svg`) - New Icon 
```
<item component="ComponentInfo{com.disney.disneyplus/com.bamtechmedia.dominguez.main.MainActivity}" drawable="disney_plus"/>
<item component="ComponentInfo{jp.co.disney.apps.gm.disneyvod/com.ascendon.disney.activity.DJLoaderActivity}" drawable="disney_plus"/>
```

### CODE REVIEW

**Comment by The-King-Slayer** [comment]
Explanation :  
-  Was really inspired by the flat abstract art style of you guys (tbh, I fell in love with it :), so I couldn't stop myself from trying it firsthand. Thus, here are my designs, and if you like any of those, I'd be glad if you add them to the pack. And anyway I'd love to hear your C&C.

Changes :
- Added 59 new icons

Note:
- It was my first time using Git and GitHub so sorry if I messed up something ;P
- YOU GUYS ROCK! I love Appstract and looking forward to more such awesome creations.

**Comment by mirrorkeydev** [comment]
Hello @The-King-Slayer!

Thank you for opening your PR! All the technical details of your PR-opening look good on my end (since you mentioned it's your first time), so without further ado I'll get straight to the icon critique. I'll group my critique into a couple of different categories, namely `good as-is`, `needs small changes`, and `needs large changes`. For all icons, I'll make sure to see if there are alternates in your alternates folder as well.

Also, I'm sorry I started getting brief with my comments at the end, I just realized that there's a lot of ground to cover and not much time.

# Overall comments
- Your color picks were consistently great!
- Good job leaving breathing room between the canvas and the icon
- Round caps on all lines were good :)
- A lot of icons lacked centering and symmetry. Make sure you know how to center things to the page and align items relative to each other.

# Good As-Is

### `adobe_creative_cloud` (alternate)
I like the alternate icon here better than the original one, and I'll explain why:

1. The alternate icon has a larger background shape that fills more room
2. The intersections of the black lines onto the shape are closer to perpendicular (blue circles). In the original, you've got a line that just barely touches the shape (green circle), as well as a line that takes its time splitting off from the shape (orange circle), if that makes sense. Perpendicular intersections look more intentional, and a line that just barely touches a shape violates one of the principles in the design ethos (see: 


### `adobe_fill_and_sign` (alternate)

I like the alternate here for many of the same reasons as I preferred the alternate for `adobe_creative_cloud`. The pencil shape simply isn't round/square enough to fill much of the background, and the line touching the tip of the pencil violates the "touch but not cross" rule.

### `adobe_xd`

Nice :) Just make sure to center shapes and keep a little more breathing room the edge with this one (for the lines and the shape).

# Needs Small Changes

### `adobe_capture`
This one looks good! The design is straightforward and simple. However, I opened up your svgs in an editor and it looks like the line thickness is off: 

If you fix the thickness to be 6px, we should be good. If you're using Inkscape, this might have happened by accident because you had this option enabled:


### `adobe_illustrator_draw`
Here you've got a small technical issue:

The 'V' is uneven and splits from the main shape on the left, but not on the right. If this was intentional, it's not intentional enough - right now, it reads as a mistake. The shape is also not [centered ](https://inkscape-manuals.readthedocs.io/en/latest/align-and-distribute.html) on the page or [symmetric ](https://indicatrix.org/post/1-2-3-mirror-modelling-in-inkscape/). Additionally, having lines run parallel to the edge of the shape isn't great (I should really mention perpendicularity in the guide).

Here's some alternate interpretations:


### `adobe_lightroom`

Make sure to center shapes on the page


Remove some of the fluff and move characters futher inwards in the shape (so they're not too close to the edges)


### `adobe_photoshop_express`

Good design, but the items aren't centered or perfect squares:

I've overlaid squares on top of the shapes in red to show you what I mean. I'd make sure to understand how to create symmetric shapes in Inkscape :)

### `adobe_photoshop_sketch` (alternate)
Still too complex and true to the original I think. You can reduce some of the shapes even further to simpler versions of themselves.

Here I used the "wave" form from the `gallery` icon and resized it a little to match the original icon's 'scribble', and changed the pencil tip to just a triangle.

# Needs Large Changes

### `adobe_connect`
Your interpretation of this one looks clean and true to the original, but I think it might stray too far from the Appstract design ethos. I know it's hard to translate some icons that have lots of small elements, but as often as possible I try to have one large unified shape in the background, which is missing here. The reason for this being that the lines will often "disappear" into the user's phone wallpaper if they have too much whitespace behind them.

I've come up with a couple of alternative ideas (your icon on the left), which the one inside the box my preferred pick.
image
It's simple and alludes to the original icon by outlining the one square that doesn't have perspective to it. I'd also be fairly happy with the rightmost one, it looks more like the icon and is still acceptably minimalist.

### `adobe_spark_post`

Too complex, way too many lines. Background shape isn't large enough/doesn't fill enough area.


### `app_500_px`

Needs renaming to `fivehundred_px` or `five_hundred_px` or `five_zero_zero_px` or similar.

Unfortunately this icon has significant issues - other-colored lines (grey in this case) are not allowed (white lines allowed for dark mode instead of black lines, but that's the only exception). There's also just too much going on in general - it's very true to the original, but it'll look really busy next to other icons.



# Conclusion

I'm deciding to end my critique here for now, because I've already sunk 1.5 hours into writing this comment and I think a lot of my comments apply to multiple icons. I hope I've given you a good idea of how to revise your icons! It pretty much boils down to: be simpler than you think you need to be, and pay close attention to centering and symmetry. If you have any questions about specific icons and other stuff, feel free to leave more comments on this PR.

File: app/xml/appfilter.xml [hunk 1]
```diff
@@ -153,7 +153,6 @@
 	<item component="ComponentInfo{com.asus.camera/com.asus.camera.CameraApp}" drawable="camera" />
 	<item component="ComponentInfo{com.cyngn.cameranext/com.android.camera.CameraLauncher}" drawable="camera"/>
 	<item component="ComponentInfo{com.cyngn.cameranextmod/com.android.camera.CameraLauncher}" drawable="camera"/>
-	<item component="ComponentInfo{com.google.android.GoogleCamera/com.android.camera.CameraLauncher}" drawable="camera" />
```
Comment (mirrorkeydev) [inline app/xml/appfilter.xml:156]: Why this removal? :0

**Comment by The-King-Slayer** [comment]
Hi Melanie,

First of all thanks a ton for going so deep into the analysis and taking
time for the critique. I loved how you explained each point with images and
markers.

But I'd like to explain a few things, if that makes sense? So just bear
with me for a min.

*1) Good As-Is*

- Regarding icons like *Adobe Creative Cloud* where you explained more
about breathing space.....I totally agree with you, understood and jorted
down. Thanks a ton for that one :)

- For *Adobe Fill and Sign*, haha actually I liked the one too for a
different reason as it continued the solid background and stroke legacy of
Adobe icons. But see that stroke problem, I kept it away firstly but then
it looked weird to me for some reason and then I went ahead and brought it
back to touch the tip. Sorry I forgot about the rule there, shouldn't have
done that XD.

*2) Needs Small Changes*

- I was extra vigilant with the stroke thickness of 6points but I see some
have escaped my eye :P That most probably was my resizing script which
scaled down the stroke thickness. Totally my bad, will fix it, thanks :D

- The *Adobe Illustrator Draw* Icon had the left stroke "wider" than the
right one and more like a stroke of V with the ink "bouncing" at the joint.
So that misalignment of "V" was more of an international design to signify
that virtual "thickness". Just see the outer edge of that V in original
icon, that's exactly the effect I wanted to create.
So if you like the perfect one, great. Otherwise, I'd like to keep this one
imperfect :)


*3) Needs Large Changes*

Agreed.
Agreed.
And Some more agreed XD

Things and design ethos you mentioned in this section were exactly WHY I
wanted the feedback from you.

"Be simpler than you think you need to be" - words to live by. Love that.

I was trying to be more true to the original while all I had to do was
Abstractify (or appstractify things :P) so your designs definitely look
cooler than mine here so I'll let you win this time hahaha.

* 4) Side Discussion*

However one small but important pointer is that I see *centering and
alignment* is a consistent issue regarding which I would like to just say
that I spent special time taking care of so I still haven't figured out for
sure where it went wrong (because obviously it did! -_-) and I'd need to
pull up the designs on the screen for that.

But still as a primary theory my prime suspects are the exact two buttons
of center align to canvas that you showed here. I think we're looking at
two different center points :|

For example take the LR icon, the center of the word LR, the backdrop and
the "shadow-line" are perfectly aligned with each other, but when you
select the whole icon, the overall center shifts a bit towards the lower
left corner where the shadow is. This *overall center* is different from
the *mutual center. *Where with mutual center I mean the center point of
backdrop square w.r.t. which the center of other elements are aligned.

So I did maintain the mutual center point but then what I did was select
all - align with center of canvas. So theoretically that made all the
difference. All that did was align the overall center (the shifted one) to
the center of the canvas instead of the mutual center -_-

However I got what you meant to say, really a huge huuuuge thanks for
pointing that out, I'd have never noticed it as that's just my natural
workflow so subconsciously that'd appear to be "normal" to me.

*Conclusion :*

Well, I can't express in words how damn happy and grateful I am for all you
did. Not trying to butter you up but really while writing this whole mail
and more while reading your one, I enjoyed it so much there was a constant
smile on my face. So really thank you from the fathom of my heart.

I've seriously taken all your advices and I'd love to fix the icons and
bring them upto the mark, as good as getting included in the Appstract icon
pack that is ;P I started all this just for fun, but now I am willing to go
beyond just to perfect my own learnings. So there's a small favour I need
from you, just send me over a list of icons that you think need small
changes and big changes.

Now now, I am in IST so I just woke up and the first thing I do is write
all this XD

Moreover it's Christmas eve, so I am just gonna go grab my coffee, take a
nice lil drive maybe get wasted a bit :P

So you too take care and enjoy the day, Merry Christmas :D

And really, thanks again for everything.

Best Regards and Greetings,
Aman

**Comment by The-King-Slayer** [comment]
Hi Melanie,

I didn't remove that, that particular piece of code shows the default cam
icon for gcam app.

I just re-linked it with the new gcam icon I made. That's all :)

Regards,
Aman

**Comment by mirrorkeydev** [comment]
Hey @The-King-Slayer!

So response to your feedback:

> But still as a primary theory my prime suspects are the exact two buttons of center align to canvas that you showed here. I think we're looking at two different center points :|

Yes, I think you're right, I probably should have clarified in the guide: when I say a design is centered, I mean that the background shape is centered. So basically, when centering, only select the background shape (not any of the lines), and then hit those two buttons. 

As a somewhat contrived example for anyone else that stumbles across this thread, take this fake icon I made up:


(I think I'll add this example to the guide as well)

In the left example, I've selected both the shape and the line, and hit center. As you noted, the median center is different than just the background center, and so the overall effect is that the entire shape looks off-center (since the background shape fills more space, it's "heavier" in the viewer's eye and makes more of a difference. In a lineup of several different icons, the goal is that the icons' background shapes are equidistant and centered on the same line). On the right, I've only centered the background shape and then manually adjusted the lines to follow. As a result, a lineup of several icons centered using the rule on the right will look more "aligned", like these from the guide:


> So there's a small favour I need from you, just send me over a list of icons that you think need small changes and big changes.

Will do, here they are:
### Already covered (above):
icons/svgs/adobe_capture.svg
icons/svgs/adobe_connect.svg
icons/svgs/adobe_creative_cloud.svg
icons/svgs/adobe_fill_and_sign.svg
icons/svgs/adobe_illustrator_draw.svg
icons/svgs/adobe_lightroom.svg
icons/svgs/adobe_photoshop_express.svg
icons/svgs/adobe_photoshop_sketch.svg
icons/svgs/adobe_spark_post.svg
icons/svgs/adobe_xd.svg
icons/svgs/app_500_px.svg

### Key:
_Which icon:_
ALT = alternate icon
MAIN = not-alternate icon

_Changes:_
CENT = center/align background shape
LTS = line tip/edge issues (touching shape without crossing)
CST = colored shapes are touching (should keep distance ideally)
BS = make background shape fill more space

### Other icons:
- icons/svgs/asphalt_9.svg - '9' line is great but background shape in ALT too complex. needs to be much simpler (in MAIN too)
- icons/svgs/behance.svg - MAIN better than ALT, maybe space out the lines a tiny bit more
- icons/svgs/bit_torrent.svg - ooo!! i like this one. only small change to center background and move lines to be more like
	image

- icons/svgs/brawl_stars.svg - both ALT and MAIN are good, except LTS. i maybe slightly prefer ALT
- icons/svgs/burger_king.svg - large changes, too complex, too many shapes, CST
- icons/svgs/camera_fv_5.svg - BS otherwise good
- icons/svgs/camera_mx.svg - ALT better because bigger background shape but CST and LTS
- icons/svgs/canva.svg - good! just CENT
- icons/svgs/chess_free.svg - wow these are beautiful (both ALT and MAIN) but unfortunately far too complex.
- icons/svgs/coursera.svg - MAIN is perfect as-is :)
- icons/svgs/crunchy_roll.svg - ALT is great, just CENT and make sure lines maintain a little more space from the edge of the background shape
- icons/svgs/dominos.svg - MAIN has shapes on top of lines, which doesn't match ethos, ALT is good but maybe offset the background shapes more/make bigger?
- icons/svgs/duolingo.svg - ALT is better but I think it could be even simpler, less traced from the original icon
- icons/svgs/edx.svg - big changes, I'd rethink. I'd probably just focus on the 'X'
- icons/svgs/filmora_go.svg - BS for both ALT/MAIN and i'd remove the little 'tail' on the blue shape
- icons/svgs/flipkart.svg - MAIN is cute! however CST and CENT. I'd probably just pick one color for this one
- icons/svgs/food_panda.svg - Both too complex, I'd rethink
- icons/svgs/gaana.svg - ALT is cute! Maybe just BS a little and watch the LTS
- icons/svgs/google_camera.svg - Love the colors but unfortunately it's too many of them. also, my google camera app is only grey and blue, so i'm not sure what you're using as reference image
- icons/svgs/ibis_paint_x.svg - Similar, too many colors and BS and CENT
- icons/svgs/idm.svg - Prefer ALT because BS, it looks great! You don't have to completely CENT for this one , you could get away with just nudging it a little to the left
- icons/svgs/idm_plus.svg - Plus should be lines and not a background shape maybe? 
- icons/svgs/infinite_design.svg - Lines too complex and CENT
- icons/svgs/infinite_painter.svg - Same as above
- icons/svgs/inshorts.svg - I like MAIN! but there's too many lines far away from the main shape, so maybe use the 'cylinder' lines shapes from ALT with a bigger backgrond shape from MAIN? no cutout holes in background shape
- icons/svgs/inshot.svg - ALT good! only thing i would critique are the 2 little triangle-shaped holes on the edge are very small, maybe make the lines square smaller so they don't exist any more
- icons/svgs/kinemaster.svg - MAIN pretty much perfect , no hole-punches from ALT please
- icons/svgs/ludo_king.svg - too many shapes and too many spikes on crown
- icons/svgs/manual_camera_geekydevs.svg - pretty good! i'd just remove every other line that overlaps with the background shape (so they're less dense)
- icons/svgs/manual_camera_lensesdevs.svg - MAIN doesn't have any breathing room on canvas sides. I'd prefer MAIN over ALT just by a little
- icons/svgs/nine_gag.svg - Wow great! Looks perfect
- icons/svgs/opera_touch.svg - Beautiful, a winner
- icons/svgs/oyo.svg - Also gorgeous
- icons/svgs/pay_tm.svg - Too many shapes and lines. If their original logo is just their words, I'd try and come up with something completely different that has the same colors and hope it's good
- icons/svgs/picsart.svg - CST and BST (although it is really pretty, just doesn't match Appstract)
- icons/svgs/pubg.svg - background shape too complex and lines too tight and close together
- icons/svgs/pubg_lite.svg - better but lines are still too complex, looks traced. go for loose shapes that hint at the original shape
- icons/svgs/rail_connect.svg - MAIN good but i'd remove the three black lines round the outside and CENT, then it'd be great. ALT has BS
- icons/svgs/remini.svg - MAIN perfect except for that I would make the black square lines a tiny bit smaller so there's not this tiny triangle overlap
	image
- icons/svgs/starbucks.svg - Both far too complex and BS
- icons/svgs/sudoku.svg - Great, A+
- icons/svgs/udemy.svg - Also wonderful
- icons/svgs/upgrad.svg - Great! Just move background shape a tiny bit to the left (doesn't have to be centered completely) and move the lines up so that you don't have LTS
- icons/svgs/utorrent.svg - Good just work on LTS and CENT
- icons/svgs/where_is_my_train.svg - MAIN good but the bottom line of the "windshield" is slightly curved, when I don't think it should be. Make circles tiny bit bigger
- icons/svgs/wynk_music.svg - Too many lines, ideally no cut-out in background shape
- icons/svgs/youtube_go.svg - Lines look beautiful and clean but don't intersect background shape enough. Maybe pick a circle or square or something else to be BS
- icons/svgs/zomato.svg - ALT far better bc BS. Honestly looks great, maybe just space out that line in the middle a little bit further from the lines surrounding it (on small screens that tiny gap will look weird).

**Comment by The-King-Slayer** [comment]
OMG Melanie.....THANKS SO MUCH for that detailed explanation on the
list....cool cool cool I got it this time.

I will fix them ASAP and get back to you in a jiffy.

REALLY thanks and best regards,
Aman

**Comment by The-King-Slayer** [comment]
Putting up a new PR with revised changes in icons

**Pull request closed without merging**